### DISCUSSION TITLE
*move* feature

### DISCUSSION THREAD

**Discussion by petesimonic** [Zettlr/Zettlr#2346] [Q&A]
Hello! Is there a way to move an item between workspaces? The problem is the files which are used in it. Usually you want your images or other things mentioned to follow the note.

So optimal behavior would be to have a list of used files, check if other notes use the file, and when transferring the note to another workspace move or copy the file. 

Such a feature could be an overkill. But anyway: **how do you manage to keep the file with the notes when actively moving items?**

**Comment by mrc-eu** [comment]
I can't believe this isn't implemented. When do I *not* want have the attachments (files, images) keep working (= no broken links) when I move a note?

**Comment by nathanlesage** [comment]
Well, that's one of the drawbacks of using Markdown: While Word or PDF documents bundle everything in one file, Markdown is plain text, and as such cannot bundle stuff. Scanning every file to see if something else changes is pretty hard. I do understand very much the need for that feature, but since Zettlr is essentially a one-man-show, I cannot give you any promises as to when I might be able to implement this.

**Reply by mrc-eu** [reply]
I personally think it shouldn't be hard if you at least support it when moving within the app. Then you only have to change the links and/or move the files for that note alone. That would be of great help.

I use a lot of screenshots (= images) in my notes. They break every time I move the note, which makes it hard to move to Zettler from Joplin. That's a shame because I really prefer Zettler.

**Reply by nathanlesage** [reply]
As I said: I totally understand, but it still doesn't make it easier to implement. There are a lot of moving parts that could break. But now that I know about this requirement I can work on something. Or, someone else who is quicker than me, since I can't promise anything.
